core/domain/entities/editor/BlockInterface.php

```diff
@@ -18,9 +19,11 @@
  * @author  Brent Christensen
  * @since   $VID:$
  */
-interface EditorBlockInterface
+interface BlockInterface
 {
 
+    const NAME_SPACE = 'event-espresso/';
```

Comment:
I'd prefer to see `eventespresso` used as the namespace.  I realize its two words but hyphens could be a problem in certain contexts in the future and we're already using `@eventespresso` for package names etc.   I definitely will _not_ push hard on this though so if you want to leave this we can do that.

Reply:
no that's fine with me. will chagne